_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+This is a step towards opening up more projects as open source code and sharing them with the outside world.
+
+### Share
+
+> All source code must be released under an appropriate OSS license.
+> We enable incidental contributions to related open standards and OSS projects.
+
+Wherever it makes sense to do so, the ESDC will share the source code of its future IT projects.
+For publication of these projects, the OSS licences recommended by GC (MIT, Apache2, GPL3, AGPL3 or LGPL3) will be preferred.
+ESDC will focus these efforts on an ESDC-centric digital government code repository.
+In addition, our developers will be free to make occasional contributions to closely related open standards and OSS projects.
+
+The principles and actions of this strategy will make it easier to obtain permission to share code with the outside world.
+
+Currently, the paperwork involved in this process takes time to complete and this holds back many of our in-house projects.
+This situation needs to change.
+
+The GC [Guide for publishing open source code](https://www.canada.ca/en/government/system/digital-government/digital-government-innovations/open-source-software/guide-for-publishing-open-source-code.html) provides recommendations how to: seek approvals, obtain rights to custom code in contracts; consider security implications; select OSS licence; select source code repository; add recommended files; publishing a legacy application.
+
+### Contribute
+
+> We strive to be an active member of the diverse open standards and OSS ecosystem.
+
+To be an active member of the diverse open standards and OSS ecosystem, we will continue to gain expertise to address product management and governance.
+ESDC will seek to become an active contributing member of key communities where it can add value, and while balancing costs.
+
+On the basis of the principles set out here, we will build and maintain lasting, meaningful relationships with open standards and OSS communities, building on experience from CEF, the EU-free and OSS auditing (EU-FOSSA/EU-FOSSA2) project and the ISA2 programme.
+This will contribute to the viability of the ecosystem and facilitate product management and governance.
+
+In addition to sharing source code, ESDC should consider joining open-source programme boards and committees, devise systematic methods for targeted funding, work to improve security and find ways to help critical open-source projects become self-sustainable.
+
+The GC [Guide for Contributing to OSS](https://www.canada.ca/en/government/system/digital-government/digital-government-innovations/open-source-software/guide-for-contributing-to-open-source-software.html) provides recommendations how to: verify OSS licence; verify contributing process and policies; additional approvals; contribute to the project; contribute through professional services; publish contributions regardless of upstream acceptance.
+
+### Secure
+
```

Comment:
Maybe a small reminder that Open source technologies play an important role in securing traffic on the internet.  OpenVPN, for instance is open source and maintain a high percentage of market share in this space.  

Open Source = Security Innovation